content/support-training.html

```diff
@@ -4,129 +4,122 @@
 ---
 
 <div class="container-fluid training">
-  
+
   <div class="row gray-bg">
-    <div class="container">
-    <div class="col-md-12 doc-content">
-      <h1>Support & Training</h1>
-      <p>This list is provided in alphabetical order. If you'd like to add your training resource, see <a href="#add">Add a resource</a>.
-    </div>
+    <div class="col-md-12">
+      <div class="container doc-content">
+        <h1>Support & Training</h1>
+        <p>This list is provided in alphabetical order. If you'd like to add your training resource, see <a href="#add">Add a resource</a>.
+      </div>
     </div>
   </div>
 
 
   <div class="row">
-    <div class="container">
-      <div class="col-md-12 doc-content training-team">
+    <div class="col-md-12">
+      <div class="container doc-content">
         <h2>Courses</h2>
-      </div>
-    </div>
-    <div class="container equal">
-      <div class="col-sm-12 col-md-6 doc-content panel-col">
-          <div class="panel panel-default">
-            <div class="panel-body">
-              <a href="https://training.robustperception.io/" target="_blank">
-                <h3>Learn how to Monitor with Prometheus</h3>
-              </a>
-              <p>Delivered by <a href="https://training.robustperception.io/" target="_blank">Robust Perception Training</a>.
-            </div>
+        <div class="row equal">
+          <div class="col-md-3 col-sm-6 doc-content tc-col">
+            <a href="https://training.robustperception.io/" class="btn btn-training btn-default btn-large" target="_blank">
```

Comment:
Hmm, wouldn't L go before R or are we sorting on the The?

Reply:
Ah yeah, fixed.